packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -597,10 +597,10 @@ export class BlobManager {
 			// If there is already an op for this storage ID, append the local ID to the list. Once any op for
 			// this storage ID is ack'd, all pending blobs for it can be resolved since the op will keep the
 			// blob alive in storage.
-			this.opsInFlight.set(response.id, [
-				...(this.opsInFlight.get(response.id) ?? []),
-				localId,
-			]);
+			this.opsInFlight.set(
+				response.id,
+				this.opsInFlight.get(response.id)?.add(localId) ?? new Set([localId]),
```

Comment:
If the localId already exists in the set, is that an error, normal, abnormal but safe to ignore, etc.?

If it's not expected we should probably assert.  If it's normal, would be good to call out under what circumstances it's normal in a code comment here.